config-bot/main

```diff
@@ -58,7 +58,7 @@ def parse_args():
 
 def load_config(fn):
     with open(fn) as f:
-        return toml.load(f)
+        return tomllib.load(f)
```

Comment:
[tommllib.load](https://pypi.org/project/tomli) requires file 'f' to be opened in binary form, with the 'rb' flag.
Solution which worked in testing:
```
return tomllib.load(f, 'rb')
```

Reply:
I think you mean `with open(fn, "rb") as f`?

Reply:
Correct :+1: